tfhe/Cargo.toml

```diff
@@ -83,6 +83,7 @@ boolean = ["dep:paste"]
 shortint = ["dep:paste"]
 integer = ["shortint", "dep:paste"]
 internal-keycache = ["lazy_static", "dep:fs2", "bincode", "dep:paste"]
+safe_serialization = ["bincode"]
```

Comment:
let's try to start using the `dep:` pattern for enabling dependencies, as specifying "bincode" also specifies the "bincode" feature IIRC, let's plan an issue PR to update the pattern, let's not do it in this PR, this is not the topic here :)